### PR TITLE
[FLINK-18199] [chinese-translation]: translate FileSystem SQL Connector page into Chinese

### PR DESCRIPTION
**Pull request by michaelli916**
## What is the purpose of the change

*Translate "Filesystem SQL Connector" page into Chinese*

## Brief change log

*Translate "Filesystem SQL Connector" page into Chinese*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.
The screen shot for the translated chinese page are as following：
image
image
image
image
image
image
image
image
image

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency):  no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive):  no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector:  no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Review by JingsongLi — COMMENTED** [review]
Thanks @michaelli916 Can you resolve the conflicts?

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 1]
```diff
@@ -24,15 +24,13 @@ specific language governing permissions and limitations
...
 
-The file system connector itself is included in Flink and does not require an additional dependency.
-A corresponding format needs to be specified for reading and writing rows from and to a file system.
+文件系统连接器本身就被包括在 Flink 中，不需要任何额外的依赖。当从文件系统中读取或向文件系统写入记录时，需要指定相应的记录格式。
 
-The file system connector allows for reading and writing from a local or distributed filesystem. A filesystem table can be defined as:
+文件系统连接器支持对本地文件系统或分布式文件系统的读取和写入。 可以通过如下方式定义文件系统表:
 
 ```sql
 CREATE TABLE MyUserTable (
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:36]: 你可以翻译下SQL中的注释吗？

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 2]
```diff
@@ -149,15 +145,14 @@ a timeout that specifies the maximum duration for which a file can be open.
   </tbody>
 </table>
 
-**NOTE:** For bulk formats (parquet, orc, avro), the rolling policy in combination with the checkpoint interval(pending files
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:152]: 可以翻译下上述的配置项的说明吗？其他配置项也都需要翻译

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 3]
```diff
@@ -149,15 +145,14 @@ a timeout that specifies the maximum duration for which a file can be open.
...
-become finished on the next checkpoint) control the size and number of these parts.
+**注意:** 对于 bulk 格式 (parquet, orc, avro), 滚动策略和检查点间隔控制了分区文件的大小和个数 (未完成的文件会在下个检查点完成）.
 
-**NOTE:** For row formats (csv, json), you can set the parameter `sink.rolling-policy.file-size` or `sink.rolling-policy.rollover-interval` in the connector properties and parameter `execution.checkpointing.interval` in flink-conf.yaml together
-if you don't want to wait a long period before observe the data exists in file system. For other formats (avro, orc), you can just set parameter `execution.checkpointing.interval` in flink-conf.yaml.
+**注意:** 对于行格式 (csv, json), 如果想使得分区文件更快地在文件系统中可见，可以设置连接器参数 `sink.rolling-policy.file-size` 或 `sink.rolling-policy.rollover-interval` ，以及 flink-conf.yaml 中的 `execution.checkpointing.interval` 。 
+对于其他格式 (avro, orc), 可以只设置 flink-conf.yaml 中的 `execution.checkpointing.interval` 。
 
-### File Compaction
+### 文件压缩
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:153]: 我觉得翻译生 “文件合并” 更好些

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 4]
```diff
@@ -184,24 +179,25 @@ The file sink supports file compactions, which allows applications to have small
...
-If enabled, file compaction will merge multiple small files into larger files based on the target file size.
-When running file compaction in production, please be aware that:
-- Only files in a single checkpoint are compacted, that is, at least the same number of files as the number of checkpoints is generated.
-- The file before merging is invisible, so the visibility of the file may be: checkpoint interval + compaction time.
-- If the compaction takes too long, it will backpressure the job and extend the time period of checkpoint.
+启用该参数后，文件压缩功能会根据设定的目标文件大小，合并多个小文件到大文件。
+当在生产环境使用文件压缩功能时，需要注意：
+- 只有检查点内部的文件才会被压缩，也就是说，至少会生成跟检查点个数一样多的文件。
+- 合并前文件是可见的，所以文件的可见性是：检查点间隔 + 压缩时长。
+- 如果压缩花费的时间很长，会对作业产生背压，延长检查点所需时间。
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:186]: 我更习惯用 “反压”

**Review by JingsongLi — CHANGES_REQUESTED** [review]
Thanks for your rebase, left some comments.

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 1 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md:36]: 好的。

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 2 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md:152]: 好的。

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 3 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md]: 嗯，“文件压缩”容易造成歧义，确实“文件合并”更好。

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 4 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md]: 好的。

**Comment by michaelli916** [comment]
Revised document based on review comments. The page screen shot are as following shows.
image
image
image
image
image
image
image
image
image

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 5]
```diff
@@ -173,35 +161,36 @@ The file sink supports file compactions, which allows applications to have small
...
     </tr>
   </tbody>
 </table>
 
-If enabled, file compaction will merge multiple small files into larger files based on the target file size.
-When running file compaction in production, please be aware that:
-- Only files in a single checkpoint are compacted, that is, at least the same number of files as the number of checkpoints is generated.
-- The file before merging is invisible, so the visibility of the file may be: checkpoint interval + compaction time.
-- If the compaction takes too long, it will backpressure the job and extend the time period of checkpoint.
+启用该参数后，文件压缩功能会根据设定的目标文件大小，合并多个小文件到大文件。
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:175]: 统一下哈，都是合并，不是压缩

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 6]
```diff
@@ -372,9 +364,9 @@ public class AnalysisCommitPolicy implements PartitionCommitPolicy {
 
 ```
 
-## Sink Parallelism
+## Sink 并行度
 
-The parallelism of writing files into external file system (including Hive) can be configured by the corresponding table option, which is supported both in streaming mode and in batch mode. By default, the parallelism is configured to being the same as the parallelism of its last upstream chained operator. When the parallelism which is different from the parallelism of the upstream parallelism is configured, the operator of writing files and the operator compacting files (if used) will apply the parallelism.
+向外部文件系统（包括 hive) 写文件时的并行度，在流处理模式和批处理模式下，都可以通过对应的 table 选项指定。默认情况下，该并行度跟上一个上游的 chained operator 的并行度一样。当配置了跟上一个上游的 chained operator 不一样的并行度时，写文件的算子和压缩文件的算子（如果使用了的话）会使用指定的并行度。
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:369]: 合并算子

Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md:369]: 好的。

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 5 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md:175]: 好的，我再检查下，统一修改为“合并”。

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 7]
```diff
@@ -279,24 +264,27 @@ Time extractors define extracting time from partition values.
...
         <td><h5>partition.time-extractor.class</h5></td>
         <td style="word-wrap: break-word;">(none)</td>
         <td>String</td>
-        <td>The extractor class for implement PartitionTimeExtractor interface.</td>
+        <td>实现了接口 PartitionTimeExtractor 的提取器类.</td>
     </tr>
     <tr>
         <td><h5>partition.time-extractor.timestamp-pattern</h5></td>
-        <td style="word-wrap: break-word;">(none)</td>
+****        <td style="word-wrap: break-word;">(none)</td>
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:277]: **** ??

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 8]
```diff
@@ -397,17 +389,21 @@ The parallelism of writing files into external file system (including Hive) can
...
   </tbody>
 </table>
 
-**NOTE:** Currently, Configuring sink parallelism is supported if and only if the changelog mode of upstream is **INSERT-ONLY**. Otherwise, exception will be thrown.
+**注意:** 当前，只有在上游的 changelog 模式是 **INSERT-ONLY** 时，才支持设置 sink 的并行度。否则的话，会抛出异常。
 
-## Full Example
+## 完整示例
 
+<<<<<<< HEAD
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:402]: Conflicts?

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 7 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md]: oops, my fault.

File: docs/content.zh/docs/connectors/table/filesystem.md (hunk 8 above)
Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md]: i will revise the whole section under "完整示例"

File: docs/content.zh/docs/connectors/table/filesystem.md [hunk 9]
```diff
@@ -217,13 +206,15 @@ To define when to commit a partition, providing partition commit trigger:
...
     </tr>
     <tr>
         <td><h5>sink.partition-commit.delay</h5></td>
         <td style="word-wrap: break-word;">0 s</td>
         <td>Duration</td>
-        <td>The partition will not commit until the delay time. If it is a daily partition, should be '1 d', if it is a hourly partition, should be '1 h'.</td>
+        <td>该延迟时间之前分区不会被提交。如果是按天的分区，应配置为 '1 d', 如果是按小时的分区，应配置为 '1 h'.</td>
     </tr>
     <tr>
         <td><h5>sink.partition-commit.watermark-time-zone</h5></td>
```
Comment (JingsongLi) [inline docs/content.zh/docs/connectors/table/filesystem.md:220]: Can you translate the description of this option too?

Comment (michaelli916) [reply docs/content.zh/docs/connectors/table/filesystem.md:220]: ok, done.

**Review by JingsongLi — APPROVED** [review]
Thanks for your update! Looks good to me.

**Pull request merged**